css/navbar.css

```diff
@@ -1,60 +1,163 @@
 nav {
-  height: 5.25rem;
-  font: 700 1.25rem 'Roboto', sans-serif;
+  font-weight: 700;
 }
 
-nav ul {
+nav li a {
+  margin: 10px;
+  display: block;
+  border-radius: 0.5rem;
+  color: #ffffff;
+  text-align: center;
+  padding: 19px 16px;
+  text-decoration: none;
+}
+
+nav li {
+  float: left;
+}
+
+nav li a:hover {
+  color: #49a82e;
+}
+
+.nav-menu {
   list-style-type: none;
   margin: 0;
   padding: 0;
   overflow: hidden;
   background-color: #1d1283;
-  height: 5.25rem;
 }
 
-nav li {
-  float: left;
+.nav-logo-li {
+  padding: 14px 20px;
 }
 
-nav li a {
-  margin: 10px 0 0 20px;
-  display: block;
-  border-radius: 0.5rem;
+.nav-logo-li a {
+  margin: 0px;
+  padding: 0px;
+}
+
+.nav-login-li {
+  float: right;
+}
+
+.nav-login-li a {
+  padding: 16px;
+}
+
+.nav-login,
+.home-tab {
+  display: none;
+}
+
+.btn-login-text {
+  font: 14px, sans-serif;
+  border-radius: 6px;
+  padding: 5px;
+  background-color: #1d1283;
+  color: white;
+  cursor: pointer;
+  border: 2px solid white;
+}
+
+.user-greet {
+  display: none;
   color: #ffffff;
-  font-weight: 700;
-  text-align: center;
-  padding: 19px 16px;
-  text-decoration: none;
+  margin: 25px;
 }
 
-nav li a:hover {
-  color: #88d870;
+.user-profile-pic {
+  width: 2em;
```

Comment:
Any specific reason for going with `em` here?

Reply:
No, we can use `px` here.